tests/integration/kubernetes/k8s-guest-pull-image-signature.bats

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env bats
+# Copyright (c) 2024 IBM Corporation
+# Copyright (c) 2024 Alibaba Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/lib.sh"
+load "${BATS_TEST_DIRNAME}/confidential_common.sh"
+load "${BATS_TEST_DIRNAME}/confidential_kbs.sh"
+
+export KBS="${KBS:-false}"
+
+setup() {
+    # TODO - do TEEs want to run these tests?
+    # if is_confidential_hardware; then
+    #     skip "Due to issues related to pull-image integration skip tests for ${KATA_HYPERVISOR}."
+    # fi
```

Comment:
We need to decide whether or not to run this on the TEE hardware, so lets run the e2e tests and make a call to delete this, or uncomment